.github/workflows/pr.yaml

```
@@ -0,0 +1,95 @@
+name: PR checks
+on: pull_request
+
+jobs:
+  run-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build image
+        run: DOCKER_BUILDKIT=1 docker build --tag docs-ubuntu-com .
+
+      - name: Run image
+        run: |
+          docker run --detach --env SECRET_KEY=insecure_secret_key --network host docs-ubuntu-com
+          sleep 1
+          curl --head --fail --retry-delay 1 --retry 30 --retry-connrefused http://localhost
+
+  run-dotrun:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dotrun
+        run: |
+          sudo snap install dotrun
+          sudo chown root:root /
+
+      - name: Install dependencies
+        run: dotrun install
+
+      - name: Build assets
+        run: |
+          sudo chown root:root /etc/gitconfig
```

Try changing this to
```suggestion
          sudo rm /etc/gitconfig
```